[SEO 14] Fix links to avoid redirects (#225)
KnapsackPro/docs.knapsackpro.com: _posts/2020-09-25-how-to-build-native-integration-with-knapsack-pro-api-to-run-tests-in-parallel-for-any-test-runner-testing-framework.md

You can rename forked project `@knapsack-pro/cypress` to `@knapsack-pro/my-test-runner` and update the info in `package.json`.

- Update name to `@knapsack-pro/my-test-runner` and `version` to `1.0.0`. [See in code](https://github.com/KnapsackPro/knapsack-pro-cypress/blob/8942e0430e9b529ab27cf877b15b2d2964f89222/package.json#L2,L3).

> - Add your test runner package to [`peerDependency`](https://docs.npmjs.com/cli/v11/configuring-npm/package-json#peerdependencies). This allows a developer to use a runner within a specified version range when the developer installs your package in their project. [See in code](https://github.com/KnapsackPro/knapsack-pro-cypress/blob/8942e0430e9b529ab27cf877b15b2d2964f89222/package.json#L62).

Reviewer: ```suggestion
- Add your package to the [`peerDependency`](https://docs.npmjs.com/cli/v11/configuring-npm/package-json#peerdependencies). This enforces your users to install a specific version of the test runner in their project alongside your package. [See in code](https://github.com/KnapsackPro/knapsack-pro-cypress/blob/8942e0430e9b529ab27cf877b15b2d2964f89222/package.json#L62).
```
